spacy_udpipe/udpipe.py

```diff
@@ -0,0 +1,202 @@
+        if self.model is None:
+            raise Exception(f"Cannot load UDPipe model from file '{path}'")
+        self._lang = lang.split("-")[0]
+        self._meta = meta or {"author": "Milan Straka & Jana Straková",
+                              "description": "UDPipe pretrained model.",
+                              "email": "straka@ufal.mff.cuni.cz",
+                              "lang": f"udpipe_{self._lang}",
+                              "license": "CC BY-NC-SA 4.0",
+                              "name": path.split("/")[-1],
+                              "parent_package": "spacy_udpipe",
+                              "pipeline": [
+                                  "Tokenizer", "Tagger", "Lemmatizer", "Parser"
+                              ],
+                              "source": "Universal Dependencies 2.5",
+                              "url": "http://ufal.mff.cuni.cz/udpipe",
+                              "version": "1.2.0"
+                              }
+
+    def __call__(
+        self,
+        text: Union[
+            str,
+            List[str],
+            List[List[str]]
+        ]
+    ) -> List[Sentence]:
+        """Tokenize, tag and parse the text and return it in an UDPipe
+        representation.
+
+        text: Input text, can be presegmented or pretokenized:
+            str             : raw text,
+            List[str]       : presegmented text,
+            List[List[str]] : pretokenized text.
+        RETURNS: Processed sentences.
+        """
+        sentences = self.tokenize(text)
+        for s in sentences:
+            self.tag(s)
+            self.parse(s)
+        return sentences
+
+    def _read(self, text: str, input_format: str) -> List[Sentence]:
+        """Convert the text to an UDPipe representation.
+
+        text: Input text.
+        input_format: Desired input format.
+        RETURNS: Processed sentences.
+        """
+        input_format.setText(text)
+        error = ProcessingError()
+        sentences = []
+
+        sentence = Sentence()
+        while input_format.nextSentence(sentence, error):
+            sentences.append(sentence)
+            sentence = Sentence()
+        if error.occurred():
+            raise Exception(error.message)
+
+        return sentences
+
+    def tokenize(
+        self,
+        text: Union[
+            str,
+            List[str],
+            List[List[str]]
+        ]
+    ) -> List[Sentence]:
+        """Tokenize input text.
+
+        text: Input text, can be presegmented or pretokenized:
+            str             : raw text,
+            List[str]       : presegmented text,
+            List[List[str]] : pretokenized text.
+        RETURNS: Processed sentences.
+        """
+        if isinstance(text, str):
+            tokenizer = self.model.newTokenizer(self.model.DEFAULT)
+        elif isinstance(text, list):
+            if isinstance(text[0], list):
+                text = "\n".join("\t".join(sent) for sent in text)
```

Comment:
Although this REALLY shouldn't happen, you could detect if there is '\t' or '\n' in one of the words the text and therefore give some warning, or comment about this in in docs.